### PR TITLE
added support for multiple gitignores files

### PR DESCRIPTION
**Pull request by r0x0d**
Added support for .gitignore creation with multiple languages. See issue #2 for more details.

### CODE REVIEW

File: file_utils.js [hunk 1]
```diff
@@ -24,18 +24,35 @@ const getGitignoreNames = async () => {
  */
 const getFileType = files => files.map(file => ({
   title: file.split('.')[0],
+  value: file.split('.')[0],
```
Comment (nicht) [inline file_utils.js]: sorry for that :(

**Comment by christopherkade** [comment]
Hello, thank you so much for this contribution ! 

Just a question:  
`gitignoreContent` contains an array of content if the user has selected multiple files, yet it doesn't seem like anything was changed in the `writeFile` method, how come?

What would be the desired behavior? A single `.gitignore` where the values are concatenated? If so it should be a simple check in the `writeFile` method.

**Comment by nicht** [comment]
Well, yes, it's a single `.gitignore` file with the content of the previous choices, so for example, if you selected the `Ada,ActionScript and Node` gitignores, it would be merged and written into a single `.gitignore` file. 

Also, i should have asked in the issue thread about that one, my bad! I assumed a merged `.gitignore` file would make more sense!

**Comment by christopherkade** [comment]
No problem at all, thank you for your contribution @nicht !

**Comment by gangadharjannu** [comment]
@nicht @christopherkade There is an issue in this implementation. `readFile` method returns Array if the user selects multiple languages otherwise it returns String. 

Because of this behaviour, if the user wants to generate gitignore for multiple languages, gitignore contents of multiple languages are separated by comma

**Pull request merged**